README.md

```diff
@@ -161,3 +161,71 @@ trainer.fit(
 ```
 
 </details>
+
+## Model Registry Mixins
+
+Lightning Models provides mixin classes that simplify pushing models to and pulling models from the registry.
+These mixins can be integrated directly into your model classes.
```

Comment:
Usually we avoid using "your" in these cases. example it can be someone else's model and user might think they can't use the mixin from third party model. 

Also would be great adding a brief explainer of what is "Mixin".